docs/Xeltalliv/simple3D.md

```diff
@@ -227,8 +228,8 @@ scale X (objectSizeX) Y (objectSizeY) Z (objectSizeZ) :: sensing
 draw mesh [my mesh] :: sensing
 end
 ```
-The extension allows you to split your large transformation into 3 separate transformations, which will be applied sequentially, allowing some of the features to do their thing inbetween.
-It also allows you to easily got between different coordinate systems using those blocks:
+The extension allows you to split your large transformation into 3 separate transformations, which will be applied sequentially, allowing some of the features to act inbetween.
+It also allows you to easily work between different coordinate systems using those blocks:
```

Comment:
I think "go", "transform" or "convert" fit better than "work". Originally it was meant to be "go".